app/views/admin/sessions/new.erb

```diff
@@ -0,0 +1,11 @@
+<h2 class="heading"><%= ts('Log in as Admin') %></h2>
```

Comment:
This page appears to have lost the HTML comments that were in the previous version (https://github.com/otwcode/otwarchive/pull/2505/files#diff-0911a8e5533f34121a28fc21f46f85a7L1). Could you please add them back?